public/style.css

```diff
@@ -101,3 +101,9 @@ button:hover {
   background-color: rgb(11, 223, 36);
   width: 0%;
 }
+#full-time{
+  font-size: 3em;
+  text-align: center;
+  font-family: Georgia, 'Times New Roman', Times, serif;
```

Comment:
can you use `font-family: 'Chakra Petch', sans-serif;`? this our font family for now.